RawEventProcessor.cpp

```diff
@@ -310,12 +316,17 @@ bool RawEventProcessor::process_syscall_event(const Event& event) {
     if (syscall_rec && syscall_field) {
         if (InterpretField(_tmp_val, syscall_rec, syscall_field, field_type_t::SYSCALL)) {
             if (starts_with(_tmp_val, S_EXECVE)) {
-                rec_type = RecordType::AUOMS_EXECVE;
-                rec_type_name = auoms_execve_name;
+                rec_type = RecordType::AUOMS_SYSCALL;
+                rec_type_name = auoms_syscall_name;
```

Comment:
The change from AUOMS_EXECVE (14688) to AUOMS_SYSCALL (10001) will break a lot of MSTIC code. Please can we revisit the decision to change this.